packages/jsonforms-renderers/src/renderers/ComputedControl.tsx

```
@@ -0,0 +1,170 @@
+        )}
+        {schema.description && (
+          <Text size="1" color="gray">
+            {schema.description}
+          </Text>
+        )}
+      </Flex>
+    </Box>
+  )
+}
+
+export default withJsonFormsControlProps(ComputedControl)
```

ESLint

[error] Fast refresh can't handle anonymous components. Add a name to your export.

(react-refresh/only-export-components)